contrib/runners/actionchainrunner/runner.yaml

```diff
@@ -0,0 +1,14 @@
+- aliases: []
+  description: A runner for launching linear action chains.
+  enabled: true
+  name: action-chain
+  runner_module: st2actions.runners.actionchainrunner
```

Comment:
We call it entry_point for other content. But if you are going to use stevedore approach, some of this would change.